test/extended/deployments/deployments.go

```diff
@@ -1030,6 +1030,8 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
 			g.By("should create ControllerRef in RCs it creates", func() {
 				dc, err = readDCFixture(simpleDeploymentFixture)
 				o.Expect(err).NotTo(o.HaveOccurred())
+				// Having more replicas will make us more resilient to pod failures
```

Comment:
Unclear why. Usually more pods means more flakes.

Reply:
The idea is that if some of them die it may cause false positive when checking if it detached or not. If 2/2 pods die, the wait condition would think that it's already detached which wouldn't be true.